src/validation/entities/schemas/signature.schema.ts

```diff
@@ -1,11 +1,13 @@
+import { parseSignaturesByType } from '@/domain/common/utils/signatures';
 import { HexBytesSchema } from '@/validation/entities/schemas/hexbytes.schema';
 
-// This does not take dynamic parts into account but we can safely
-// apply it to proposed signatures as we do not support contract
-// signatures in the interface
-function isSignature(value: `0x${string}`): boolean {
-  // We accept proposals of singular or concatenated signatures
-  return (value.length - 2) % 130 === 0;
+function isSignature(vlaue: `0x${string}`): boolean {
```

Comment:
Nit: `value: `0x${string}`

Reply:
Fixed in 74332901f1a21437d37173f2813dc4c2fb1e3433.